twopaneview/examples/README.md

```diff
@@ -0,0 +1,51 @@
+# TwoPaneView
+
+## Status
+
+- Android:
+  - 10+
+- react-native:
+  - supported versions "<strong>&gt;= 0.60.5</strong>"
+
+## API
+
+[Documentation](https://github.com/react-native-community/discussions-and-proposals/issues/197)
+- `Auto` 
+![](https://github.com/microsoft/react-native-dualscreen/raw/master/twopaneview/examples/paneModesDemo/auto.JPG)
+- `Single`
+![](https://github.com/microsoft/react-native-dualscreen/raw/master/twopaneview/examples/paneModesDemo/single.JPG)
+- `Dual`
+![](https://github.com/microsoft/react-native-dualscreen/raw/master/twopaneview/examples/paneModesDemo/dual.JPG)
+- `panePriority='pane2'`
+![](https://github.com/microsoft/react-native-dualscreen/raw/master/twopaneview/examples/paneModesDemo/priority2.JPG)
+- `onModeChanged={()=>{console.log('onModeChanged');}`
+
+## Installation
+
+### Please refer to the react-native-dualscreen page to install the latest version of DualScreenInfo first.
+
+- Current examples work by adding the following dependency in  `package.json`:
+
+  ```diff
+  + "react-native-dualscreen": "file:../dualscreeninfo",
+  ```
+- Then `import TwoPaneView from '../TwoPaneView'`
```

Comment:
Needs updating to reflect npm dependencies

Reply:
Right should be an npm package